routing/router.go

```diff
@@ -435,6 +423,29 @@ func (r *ChannelRouter) Start() error {
 		}
 	}
 
+	// If AssumeChannelValid is present, then we'll only prune channels once
+	// we detect them as zombies.
```

Comment:
comment doesn't seem to match code, this looks like we prune them immediately

Reply:
i see now, would update comment to say if assume channel valid is present, we won't start a chain view to track spentness or something

Reply:
Fixed.